curl.spec

```diff
@@ -11,7 +11,7 @@ Requires: zlib
 export OPENSSL_ROOT
 export ZLIB_ROOT
 case %cmsplatf in
-  slc6*) KERBEROS_ROOT=/usr ;;
+  slc7*|slc6*) KERBEROS_ROOT=/usr ;;
```

Comment:
Since we do not support slc5 anymore, I would drop its stanza and make the slc6 / slc7 one default.